pkg/healthcheck/healthcheck.go

```diff
@@ -154,6 +154,8 @@ const (
 	certKeyName                   = "tls.crt"
 	keyOldKeyName                 = "key.pem"
 	keyKeyName                    = "tls.key"
+
+	podStatusRunning = "Running"
```

Comment:
This was added because `bin/lint` picked up 3 occurrences of the string in this file after this change.

Reply:
I think you can use `corev1.PodRunning`